inventory_management/inventory_management/doctype/item/item.json

```diff
@@ -0,0 +1,114 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "autoname": "format:{item_code}",
+ "creation": "2024-01-11 15:09:04.062308",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "details_tab",
+  "section_break_hh6o",
+  "column_break_tdxp",
+  "item_code",
+  "item_name",
+  "description",
+  "column_break_ivkf",
+  "opening_warehouse",
+  "opening_qty",
+  "opening_valuation_rate",
+  "tab_2_tab"
+ ],
+ "fields": [
+  {
+   "fieldname": "section_break_hh6o",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "item_code",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Code",
+   "reqd": 1,
+   "unique": 1
+  },
+  {
+   "fieldname": "opening_warehouse",
+   "fieldtype": "Link",
+   "label": "Opening Warehouse",
+   "mandatory_depends_on": "doc.opening_qty || doc.opening_valuation_rate;",
+   "options": "Warehouse",
+   "reqd": 1,
+   "set_only_once": 1
+  },
+  {
+   "fieldname": "opening_qty",
+   "fieldtype": "Int",
+   "label": "Opening Qty",
+   "mandatory_depends_on": "doc.opening_warehouse || doc.opening_valuation_rate;",
+   "set_only_once": 1
+  },
+  {
+   "fieldname": "opening_valuation_rate",
+   "fieldtype": "Currency",
+   "label": "Opening Valuation Rate",
+   "mandatory_depends_on": "doc.opening_warehouse || doc.opening_qty;",
+   "set_only_once": 1
+  },
+  {
+   "fieldname": "description",
+   "fieldtype": "Data",
```

Comment:
Description field should be Small Text/Long Text. Move this to a separate section. Implement suggestions from the last review: https://docs.google.com/document/d/1TZ6zm1tSEcpy5rmVtFnCV-BkL937sHtVXcrGNH3dGRI/edit#heading=h.po2qk7ka7b91

Add workspace too